pkg/streamingpromql/functions.go

```diff
@@ -96,6 +96,23 @@ func AbsentFunctionOperatorFactory(args []types.Operator, memoryConsumptionTrack
 	return o, nil
 }
 
+func AbsentOverTimeFunctionOperatorFactory(args []types.Operator, memoryConsumptionTracker *limiting.MemoryConsumptionTracker, _ *annotations.Annotations, expressionPosition posrange.PositionRange, timeRange types.QueryTimeRange, argExpressions parser.Expressions) (types.InstantVectorOperator, error) {
+	functionName := "absent_over_time"
+	if len(args) != 1 && len(argExpressions) != 1 {
```

Comment:
Should we fail if either does not contain exactly one element?

```suggestion
	if len(args) != 1 || len(argExpressions) != 1 {
```